docs/3-retrieving-data-as-lists.md

```diff
@@ -54,21 +60,23 @@ const queryList = af.database.list('/items', {
 To get the list in realtime, create a list binding as a property of your component or service.
 Then in your template, you can use the `async` pipe to unwrap the binding.
 
+Update `/src/app/app.component.ts` to import `FirebaseListObservable` from angularfire2 and iterate thru the list once data is retrieved. Also note the change in attribute `templateUrl` to inline `template` below.
```

Comment:
Why are we changing this to an inline template? If the angular-cli installation creates it as an external URL, I'd prefer to use that convention as it's clearly our recommended best practice. @deast opinions?